app/services/user_importer.rb

```diff
@@ -0,0 +1,16 @@
+require "csv"
+
+class UserImporter
+  def call(csv_path)
+    @csv_path = csv_path
+    upsert_users
+  end
+
+  private def upsert_users
```

Comment:
I wonder if we should wrap this in a transaction so that if there is a failure, we roll back?

https://api.rubyonrails.org/classes/ActiveRecord/Transactions/ClassMethods.html